docs/index.md

```diff
@@ -86,6 +86,8 @@ Installing the latest prometheus plugin will create a config file (`~/.steampipe
 ```hcl
 connection "prometheus" {
   plugin = "prometheus"
+
+  # The address of your Prometheus (ignore if PROMETHEUS_URL env var is set).
```

Comment:
Can you please update this so it's consistent with the wording in `config/prometheus.spc`?